natalie/README.md

```diff
@@ -0,0 +1,227 @@
+### Homework #1
+(Due Monday 7/20)
+
+You need to turn in this homework by cloning this repo, making a new branch with your solutions, pushing to github, and issuing a pull request.
+For details, follow the instructions [here](http://portlandcodeschool.github.io/jse/2015/01/07/command-line-and-git-slides/#/14).
+
+Please also read the [learning objectives](objectives.md) for this week.
+
+---
+
+**1)** (_30% of total time_)
+Predict the output of each of the following expressions, and explain the rules which determine that answer.  (_Hint: The rules for operator `==` are complicated, but do your best!_)
+
+If the output depends on the value or type of variable _x_, identify the conditions (what _x_ is) when the expression will output true (or false, if that's simpler).  Assume the cases are independent, and _x_ is reset to an unknown value before each.
+
+Some of these are tricky!  Don't trust your first instinct.  
+
+
+**a)** `"1" == 1`
+
+true - "1" is kind of like 1
+
+**b)** `"1" === 1`
+
+false - the string "1" is not exactly the same as the number 1
+
+**c)** `x == 'x'`
+
+false, unless the value of _x_ is 'x'; 'x' will stay 'x' because it is a string; the value of _x_ will not be inserted (e.g. 'banana' == 'x' is false)
+
+**d)** `x == (x+'')`
+
+true, unless the value of _x_ is a boolean value - if _x_ is a number, like 1, then this equates to 1 == "1" and the number one is kind of like the string '1'.  If _x_ is a string, then both sides would be exactly alike (e.g. "banana" == "banana").  If _x_ is a boolean value, then you would get something like
+true == "true" and the boolean value of true is not kind of like the string "true".
+
+**e)** `'' == ' '`
+
+false - I had initially thought this would be true because they are both strings but I guess the left side is just empty while the right side is a string....not completely confident on this one.
```

Comment:
You are reading this correctly! An empty string: '' is not the same as ' '. That space makes all the difference.